ofnetMaster.go

```diff
@@ -174,7 +175,7 @@ func (self *OfnetMaster) EndpointAdd(ep *OfnetEndpoint, ret *bool) error {
 			err := client.Call("OfnetAgent.EndpointAdd", ep, &resp)
 			if err != nil {
 				log.Errorf("Error adding endpoint to %s. Err: %v", node.HostAddr, err)
-				return err
+				// Continue sending the message to other nodes
```

Comment:
I'm assuming RPC error happens when node has truly gone away. When node comes back online, we have a sync mechanism. If we are seeing intermittent RPC failures, lets debug those. This layer largely assumes transport is reliable.